packages/flame/test/components/sprite_animation_group_component_test.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flame/components.dart';
+import 'package:test/test.dart';
+
+import '../util/mock_image.dart';
+
+enum AnimationState {
+  idle,
+  running,
+}
+
+void main() async {
+  // Generate a image
+  final image = await generateImage();
+
+  group('SpriteAnimationGroupComponent test', () {
```

Comment:
Maybe a test when `removeOnFinish` is used too?

Reply:
This method is already tested on the parent class, I am not sure if it worth re creating those tests for this class which don't adds any new behavior to that method.

Or do you see any cases that I may be missing?